config/development.yaml

```diff
@@ -14,6 +14,10 @@ services:
     clientSecret: 'NEED TO INSERT SECRET'
     systemUsername: 'dummy_user'
     systemPassword: 'dummy_password'
+  case:
+    #url: 'http://localhost:4452'
+    url: 'http://ccd-data-store-api-prl-ccd-definitions-pr-427.service.core-compute-preview.internal'
```

Comment:
default and dev is pointing to seperate CCD data store pr.

Reply:
Fixed in latest build